framework/Metrics/metrics/MetricUtilities.py

```diff
@@ -139,7 +140,8 @@ def _getPDFandCDFfromWeightedData(data, weights, numBins, uniformBins, interpola
     fPrimeData[i] = fPrime
   pdfFunc = mathUtils.createInterp(midpoints, fPrimeData, 0.0, 0.0, interpolation)
   mean = np.average(data, weights = weights)
-  dataStats = {"mean":mean,"minBinSize":minBinSize,"low":low,"high":high}
+  std = np.std(data)
+  dataStats = {"mean":mean,"std":std,"minBinSize":minBinSize,"low":low,"high":high}
```

Comment:
for consistency, please use "stdev" instead of "std" for the key, or update function "_convertToCommonFormat" to use "std" instead of "stdev".

Reply:
Changed to "stdev".

Reply:
I am not sure whether it is appropriate to use np.std here, as this function seems to work with weighted data.